python/tests/unit/arcticdb/version_store/test_aggregation.py

```diff
@@ -137,6 +138,94 @@ def test_hypothesis_max_min_agg(lmdb_version_store, df):
     assert_frame_equal(expected, vit.data)
 
 
+def count_aggregation(lmdb_version_store, df):
+    lib = lmdb_version_store
+    assume(not df.empty)
+
+    q = QueryBuilder()
+    q = q.groupby("grouping_column").agg({"a": "count"})
+    expected = df.groupby("grouping_column").agg({"a": "count"})
+    expected.replace(
+        np.nan, np.inf, inplace=True
+    )  # New version of pandas treats values which exceeds limits as np.nan rather than np.inf, as in old version and arcticdb
+    expected = expected.astype(np.uint64)
+
+    symbol = "count_agg"
+    lib.write(symbol, df)
+    vit = lib.read(symbol, query_builder=q)
+    vit.data.sort_index(inplace=True)
+
+    assert_frame_equal(expected, vit.data)
+
+
+@use_of_function_scoped_fixtures_in_hypothesis_checked
+@settings(deadline=None)
+@given(
+    df=data_frames(
+        [
+            column("grouping_column", elements=string_strategy, fill=string_strategy),
+            column("a", elements=non_zero_numeric_type_strategies()),
+        ],
+        index=range_indexes(),
+    )
+)
+def test_hypothesis_count_agg_non_zero_numeric(lmdb_version_store, df):
```

Comment:
We can remove this and the equivalent dynamic schema test. We only need `non_zero_numeric_type_strategies` to avoid dividing by zero in the filtering and projection tests.

Reply:
Should we replace the `non_zero_numeric_type_strategies` with `numeric_type_strategies` in `mean`, `sum` and `min/max` tests as well? I can do that in another PR since it's unrelated to the `count`.

Reply:
Yes, we can just use `numeric_type_strategies ` for all of them since there's no division by the column values involved.